src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et3ResponseService.java

```diff
@@ -41,6 +39,8 @@
 public class Et3ResponseService {
 
     public static final String ET3_ATTACHMENT = "ET3 Attachment";
+    public static final String RESPONSE_TO_A_CLAIM = "Response to a Claim";
```

Comment:
can be removed as already exists in et-common
https://github.com/hmcts/et-common/blob/master/src/main/java/uk/gov/hmcts/ecm/common/model/helper/DocumentConstants.java#L10

Reply:
fixed